pallets/automation-time/src/mock.rs

```diff
@@ -71,14 +72,21 @@ impl pallet_timestamp::Config for Test {
 
 parameter_types! {
 	pub const MaxTasksPerSlot: u32 = 2;
+	pub const MaxBlockWeight: Weight = 600_000;
```

Comment:
where'd we get this?

Reply:
I just need a value to model test cases after